5.5/apache/Dockerfile

```diff
@@ -27,6 +27,15 @@ RUN a2enmod rewrite headers
 COPY magento.conf /etc/apache2/conf-enabled/
 
 
+ENV ION_CUBE_PHP_VERSION "5.5"
+ENV PHP_EXTENSION_DIR "/usr/local/lib/php/extensions/no-debug-non-zts-20121212"
```

Comment:
Regarding my suggestion not working: http://stackoverflow.com/questions/10856129/setting-an-environment-variable-before-a-command-in-bash-not-working-for-second